flow/connectors/utils/aws.go

```diff
@@ -85,13 +86,15 @@ func GetClickhouseAWSSecrets(bucketPathSuffix string) *ClickhouseS3Credentials {
 	awsKey := os.Getenv("PEERDB_CLICKHOUSE_AWS_CREDENTIALS_AWS_ACCESS_KEY_ID")
 	awsSecret := os.Getenv("PEERDB_CLICKHOUSE_AWS_CREDENTIALS_AWS_SECRET_ACCESS_KEY")
 	awsBucketName := os.Getenv("PEERDB_CLICKHOUSE_AWS_S3_BUCKET_NAME")
+	awsEndpoint := os.Getenv("PEERDB_CLICKHOUSE_AWS_S3_ENDPOINT")
```

Comment:
Can you change this to Also can you use `PEERDB_CLICKHOUSE_AWS_ENDPOINT_URL_<SERVICE>` as the suffix instead of `PEERDB_CLICKHOUSE_AWS_S3_ENDPOINT `, would allow us to be in sync with 
 https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-endpoints.html#endpoints-service-specific-table

Reply:
having `<` and `>` in env name is a bit unconventional. maybe 
`PEERDB_CLICKHOUSE_AWS_ENDPOINT_URL_SERVICE` ?

Reply:
> having `<` and `>` in env name is a bit unconventional. maybe `PEERDB_CLICKHOUSE_AWS_ENDPOINT_URL_SERVICE` ?

Yes by <SERVICE> I mean `PEERDB_CLICKHOUSE_AWS_ENDPOINT_URL_S3`

check the url for more clarity

Reply:
As in one environment variable for each service?

Reply:
just the one for s3 for now